charts/scaffold/README.md

```diff
@@ -0,0 +1,77 @@
+# scaffold
+
+Umbrella chart for deploying the components of the [sigstore](https://www.sigstore.dev/) architecture.
+
+The following components are also included as either direct components or through chart dependencies:
+
+* ctlog
```

Comment:
Perhaps it is worthy to add a link to https://github.com/google/certificate-transparency-go here.

Reply:
Added